static/js/base.js

```diff
@@ -124,4 +139,5 @@ function log(text) {
         }
     }
     logged = document.getElementById("consoleLog").innerText += text + "\n";
+    document.getElementById('consoleLog').scrollTop = document.getElementById('consoleLog').scrollHeight;
```

Comment:
Maybe use a variable instead of repeating `document.getElementById("consoleLog")`?